python/lsst/utils/packages.py

```diff
@@ -101,6 +102,30 @@ def getVersionFromPythonModule(module: types.ModuleType) -> str:
     return str(version)
 
 
+@lru_cache(maxsize=1)
+def getAllPythonDistributions() -> dict[str, str]:
+    """Get the versions for all Python distributions that are installed.
+
+    Returns
+    -------
+    packages : `dict`
+        Keys (type `str`) are distribution names; values (type `str`) are their
```

Comment:
Per [the dev guide](https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-dict):
```suggestion
    packages : `dict` [`str`, `str`]
        Keys are distribution names; values are their
```

Reply:
Yes. Copy and paste from another location that used the old style. Will fix.